server/models/StageModel.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose')
+
+const stageSchema = new mongoose.Schema({
+  _id: {
+    type: String,
+    required: true,
+  },
+  dayId: {
+    type: String,
+    required: true,
+  },
+  tripId: {
+    type: String,
+    required: true,
+  },
+  dayIndex: {
+    type: Number,
+    required: true,
+  },
+  stageIndex: {
+    type: Number,
+    required: true,
+  },
+  stagesLongitude: Number,
+  stagesLatitude: Number,
+  destination: String,
+  desription: String,
+  colourOfPin: Number,
```

Comment:
I like how you guys did this as a number bc I was just thinking the same thing. does this bring back the merit of day view? since we are storing the same pinColor for every pin in a day

Reply:
also can we just call this "colorId" or "colorNumber" since the teacups also use this colour

Reply:
Sure, fixed!